reports/inventory/helpers.ts

```diff
@@ -146,7 +159,11 @@ export function getStockBalanceEntries(
     .flat();
 }
 
-function getSBE(item: string, location: string): StockBalanceEntry {
+function getSBE(
```

Comment:
Why have you added `batchNumber` to the function if you haven't used it in the body?

Reply:
corrected